# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/chrony/chrony_devfs.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Naur chrony-1.20.orig/conf.c chrony-1.20/conf.c
--- chrony-1.20.orig/conf.c	2003-09-22 22:22:30.000000000 +0100
+++ chrony-1.20/conf.c	2003-10-04 22:43:53.000000000 +0100
@@ -96,7 +96,7 @@
 /* ================================================== */
 /* Configuration variables */
 
-static char *rtc_device = "/dev/rtc";
+static char *rtc_device = "/dev/misc/rtc";
 static int acquisition_port = 0; /* 0 means let kernel choose port */
 static int ntp_port = 123;
 static char *keys_file = NULL;
